.github/workflows/publish.yml

```
@@ -54,6 +54,14 @@ jobs:
     - uses: actions/checkout@v2
       with:
         submodules: true
+
+    - name: Setup Node.js environment
+      env:
+        ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'
+      uses: actions/setup-node@v1.4.3
+
+    - name: build SpeckleUiApp
+      run: cd SpeckleUiApp; npm install; npm run build      
```

It's best to split them up so you can see where the failure is.

```suggestion
    - name: Install third party dependencies
      working-directory: SpeckleUiApp
      run: npm ci 
      
    - name: build Speckle UI App
      working-directory: SpeckleUiApp
      run: npm run build 
```